<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElepticalInterface</class>
 <widget class="QMainWindow" name="ElepticalInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ElepticalInterface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Solve</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Epsilon</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit_eps">
      <property name="text">
       <string>0.1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignLeft">
     <widget class="QLineEdit" name="lineEdit_N">
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Libman</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>Analytic</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>Zeydel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Relaxation method</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>N</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
